README.md

```diff
@@ -1,20 +1,44 @@
 [![code style: prettier](https://img.shields.io/badge/code_style-prettier-ff69b4.svg?style=flat-square)](https://github.com/prettier/prettier)
 
```

Comment:
I would add a heading here, inline with what we have for the [Integration](https://github.com/madetech/mca-beacons-integration) and [Spring Boot Service](https://github.com/madetech/mca-beacons-service).  So something along the lines of:

```md
# Beacons Registration Web Application
```

Or other words that are more appropriate